src/Traits/ItalianHolidays.php

```diff
@@ -0,0 +1,106 @@
+<?php
+
+namespace CarbonMacros\Traits;
+
+use Illuminate\Support\Carbon;
+
+trait ItalianHolidays
+{
+    public function registerItalianHolidays()
```

Comment:
This looks really great. Thank you for adding the links to external sources referencing the dates